<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpiniojnForm</class>
 <widget class="QWidget" name="OpiniojnForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QDateEdit" name="dateEdit_BithDate"/>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_Apply">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_Name">
      <property name="placeholderText">
       <string>Patient Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Birthday:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QRadioButton" name="radioButton_MakeAppoitnment">
      <property name="text">
       <string>Make Appoitment</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QPlainTextEdit" name="plainTextEdit_Opinion">
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_Cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
